src/Components/Button.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import styled from "styled-components";
+
+const Button = ({
+  onClick,
+  children,
+  subText,
+  bgcolor,
+  color,
+  width,
+  weight,
+}) => {
+  return (
+    <ButtonStyle
+      onClick={onClick}
+      bgcolor={bgcolor}
+      color={color}
+      width={width}
+      weight={weight}
+    >
+      <span className="text">{children}</span>
+      <p clasName="subtext">{subText}</p>
+    </ButtonStyle>
+  );
+};
```

Comment:
```suggestion
const Button = ({ children, subText, ...stylesProps}) => {
  return (
    <ButtonStyle {...stylesProps}>
      <Text>{children}</Text>
      <SubText>{subText}</SubText>
    </ButtonStyle>
  );
};

const Text = styled.span``
const SubText = styled.p``
```
- 위와 같이 작성할 수 있습니다
- `clasName` 오타입니다